src/main/kotlin/model/Delius.kt

```diff
@@ -57,6 +57,11 @@ class Delius private constructor() {
       IM.system.uses(system, "pushes service user contact information to", "IAPS")
       ProbationPractitioners.crc.uses(system, "records and reviews assessment decision, sentence plan in")
       ProbationPractitioners.nps.uses(system, "records and reviews assessment decision, sentence plan, pre-sentence report, referrals in")
+      CourtUsers.courtAdministrator.uses(system, "records CAS decision, referrals in")
+
+      OASys.system.uses(system, "assessment info, risk measures are copied into", "NDH")
+      NOMIS.system.uses(system, "offender data is coped into", "NDH")
+      NOMIS.system.uses(system, "offender data is copied into", "NDH")
```

Comment:
I'm still a little uneasy about defining relationships this way round. It feels like the relationship should be defined alongside the initiator of the communication, so the class is more explicit about the outgoing connections of that system. To be fair, it makes no difference to the generated diagrams, but I think it makes it easier to build a mental model from the code.

Again, this looks fine for this PR. Maybe we could decide/record this as an ADR?! 🤯

Reply:
👍 Raised for discussion in #51